RockWeb/Themes/NewSpring/assets/Lava/LAYOUT/search-results.lava

```diff
@@ -1,15 +1,18 @@
 <div id="results" class="row text-center">
-    
+
 </div>
 
+{% comment %}Set search engine id to use from site attribute value{% endcomment %}
+{% assign gcseid = CurrentPage.Layout.Site | Attribute:'GoogleCustomSearchEngineId' %}
+
 <script>
 document.addEventListener("DOMContentLoaded", function() {
 
     var initialResultsHTML = `
     <div class="col-md-12">
     {[ section title:'Not sure what to search? Start here!' ]}
         <p style="line-height: 41px;">
-            {% definedvalue where:'DefinedTypeId == 103' sort:'Order' iterator:'values' %}
+            {% definedvalue where:'DefinedTypeId == {{ suggestionsdefinedtypeid }}' sort:'Order' iterator:'values' %}
```

Comment:
I'm guessing these are being passed in now?

Reply:
Yep. Enables us to have separate suggestions per use-case.

Reply:
👍